wkcuber/api/Layer.py

```diff
@@ -0,0 +1,142 @@
+from shutil import rmtree
+from os import mkdir
+from os.path import join
+import logging
+from wkw import wkw
+
+from wkcuber.api.MagDataset import MagDataset, WKMagDataset, TiffMagDataset
+from wkcuber.mag import Mag
+
+
+class Layer:
+    def __init__(self, name, dataset, dtype, num_channels):
+        self.name = name
+        self.dataset = dataset
+        self.dtype = dtype
+        self.num_channels = num_channels
+        self.mags = {}
+
+        full_path = join(dataset.path, name)
+        try:
+            mkdir(full_path)
```

Comment:
Will this raise an error (which is caught and printed then), when the layer already exists? When opening an existing dataset, it would be expected that the folder exists, right? Maybe `makedirs(..., exist_ok=True)` would make sense?